core/jazz_ui/src/app/secondary-components/advanced-filters/internal/advanced-filters.component.ts

```diff
@@ -90,11 +87,31 @@ export class AdvancedFiltersComponent implements OnInit {
     }
     setSlider(event){
         this.sliderMax=event;
+        var timePeriodList = [];
+        for (var i = this.sliderFrom; i <= this.sliderMax; i ++){
```

Comment:
why not map

Reply:
Removed setSlider() since the slider component is removed.